schemas/deviceToCloud/gnss/gnss.json

```diff
@@ -0,0 +1,100 @@
+{
+   "title": "GNSS",
+   "description": "GNSS device coordinate data",
+   "type": "object",
+   "oneOf": [
+      {
+         "properties": {
+            "appId": {
+               "type": "string",
+               "const": "GNSS"
+            },
+            "messageType": {
+               "type": "string",
+               "const": "DATA"
+            },
+            "data": {
+               "type": "string",
+               "description": "NEMA string",
+               "pattern": "^\\$(GPGGA|GPGLL|GPGSA|GPGSV|GPRMC|GPVTG)(,[0-9A-Z\\*\\.]*)+\n?$"
```

Comment:
**question:** not all sentences contain location data. Is that intentional to allow them here?

Reply:
Small typo, "NEMA string" --> "NMEA string"